lib/WWW/OneAll.pm

```diff
@@ -56,11 +56,16 @@ sub request {
 
     $errstr = '';    # reset
 
+    # Format the query params if exist
+    my $ref_url_params = delete $params{query_params};
```

Comment:
Is this part invoked by `sub _delete_user`? If yes, is `$params{query_params} = ["confirm_deletion=true"]` ? Is there any other place in codebase which invoke this subroutine ?

Reply:
Yes. Only the `_delete_user` will invoke this part. 
The request() subroutine is called in these places in the code.
https://github.com/binary-com/perl-WWW-OneAll/pull/6/files#diff-42cd356d019e3293c13ff181391680620de4571a664f7509e7d5f3c8eab8c6e9R44-R52